tools/deployment/presto-clp/scripts/set-up-config.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+
+cUsage="Usage: ${BASH_SOURCE[0]} <clp-package-dir>"
+if [ "$#" -lt 1 ]; then
+    echo "$cUsage"
+    exit
+fi
+clp_package_dir=$1
+
+venv_dir=${script_dir}/.venv
+if [ ! -d "${venv_dir}" ]; then
+    echo "Setting up Python venv in '${venv_dir}'..."
+    python3 -m venv "${script_dir}/.venv"
+fi
+source "${script_dir}/.venv/bin/activate"
+
+echo "Installing required Python packages..."
+pip3 install -r "${script_dir}/requirements.txt"
+
+echo "Generating environment variables file for user-configured properties..."
+python3 "${script_dir}/generate-user-env-vars-file.py" \
+    --clp-package-dir "${clp_package_dir}" \
+    --output-file "${script_dir}/../.env"
```

Comment:
the `.env` file should be added to https://github.com/y-scope/clp/blob/main/tools/deployment/presto-clp/.gitignore 

@coderabbitai create an issue to track